sdk/python/feast/infra/online_stores/contrib/postgres.py

```diff
@@ -13,26 +14,28 @@
 from feast.feature_view import FeatureView
 from feast.infra.key_encoding_utils import serialize_entity_key
 from feast.infra.online_stores.online_store import OnlineStore
-from feast.infra.utils.postgres.connection_utils import _get_conn
+from feast.infra.utils.postgres.connection_utils import _get_connection_pool
```

Comment:
We are forcing `pool` connection **by default**, we should leave it configurable by the user (he should choose the connection time that will be used), force `pool` connection in small project will be *"expensive"* (unnecessary resource usage).

**My suggestion** would be to let the user **choose** the connection type, **not `pool`** by *default* and the projects that need pool define it as pool